lib/squake/client.rb

```diff
@@ -121,7 +122,7 @@ def call(path:, headers: {}, method: :get, params: nil, api_base: nil, api_key:
       ::Oj.load(result_body, symbol_keys: true)
     rescue ::Oj::ParseError, TypeError, JSON::ParserError, EncodingError => e
       # in case of an error, Squake's response body is HTML not JSON
-      { 'error' => { 'message' => e.message, 'body' => result_body } }
+      { error: { 'message' => e.message, 'body' => result_body } }
```

Comment:
This is the actual error here, right? Before we returned a hash with string keys, but the signature defines symbol keys.

Reply:
not this but the one with `Array(response.body)`, but this one I found along the way :)